src/libraries/System.Data.OleDb/src/ArchitectureSpecificHelpers.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
```

Comment:
Missing license header

Reply:
Deleted file and moved code to existing files.